crates/tauri-bundler/src/bundle/linux/debian.rs

```diff
@@ -133,8 +133,8 @@ pub fn generate_data(
 fn generate_changelog_file(settings: &Settings, data_dir: &Path) -> crate::Result<()> {
   if let Some(changelog_src_path) = &settings.deb().changelog {
     let mut src_file = File::open(changelog_src_path)?;
-    let bin_name = settings.main_binary_name();
```

Comment:
let's also document that the changelog path changed

Reply:
for tauri v1, the path is unchanged, it is only a breaking change for those on v2. Added a change file nevertheless